xbmc/utils/SysfsUtils.h

```diff
@@ -13,10 +13,11 @@
 class SysfsUtils
 {
 public:
-  static int SetString(const std::string& path, const std::string& valstr);
-  static int GetString(const std::string& path, std::string& valstr);
-  static int SetInt(const std::string& path, const int val);
-  static int GetInt(const std::string& path, int& val);
+  static bool SetString(const std::string& path, const std::string& value);
+  static const std::string GetString(const std::string& path);
+  static bool SetInt(const std::string& path, const int value);
+  static const int GetInt(const std::string& path);
+
   static bool Has(const std::string& path);
```

Comment:
Are these helpers actually useful? Why don't you just try to read the file (which the code will do after using this function anyway)?